package.json

```diff
@@ -16,20 +17,14 @@
     "create-minor": "sh ./scripts/release minor",
     "create-patch": "sh ./scripts/release patch",
     "publish-release": "npm test && sh ./scripts/publish",
-    "lint": "eslint src",
-    "format": "prettier --config .prettierrc.js --write \"src/**/*.{js,json,md}\" \"docs/**/*.{js,json,md}\" \"*.{js,json,md}\""
+    "lint": "eslint src"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
-  "lint-staged": {
-    "**/*.{js,css,md}": [
-      "npm run format",
-      "git add"
-    ]
-  },
+  "lint-staged": "npm run lint",
```

Comment:
A major concern here: we are using `lint-staged` when we are committing in git. `npm run format` will use prettier to automatically fix format for the changed files. I'd prefer to keep that behavior.

Also, `"npm run lint"` is not a valid config for `lint-staged`. You will see the following warning when you are committing:

```
husky > pre-commit (node v8.16.0)
● Validation Warning:

  Unknown option "0" with value 'n' was found in the config root.
...

Please refer to https://github.com/okonet/lint-staged#configuration for more information...
No staged files match any of provided globs.
```

Reply:
If prettier is blocking vscode from fixing your code on save, let's try to fix that instead of removing prettier.

Reply:
I'll add them back